influence_benchmark/generate_histories/generate_histories.py

```diff
@@ -212,14 +208,7 @@ async def main():
         save_dict["histories"] = combined_histories
 
         with open(file_name, "w") as f:
-            yaml.dump(
-                save_dict,
-                f,
-                default_flow_style=False,
-                allow_unicode=True,
-                Dumper=BlockDumper,
-                sort_keys=False,
-            )
+            json.dump(save_dict, f, indent=2, ensure_ascii=False)
```

Comment:
ok so we've just killed the yaml dumping entirely?

Reply:
yeah